Makefile

```diff
@@ -159,25 +159,29 @@ $(SCOPE_UI_TOOLCHAIN_UPTODATE): client/yarn.lock $(SCOPE_UI_BUILD_UPTODATE)
 	touch $(SCOPE_UI_TOOLCHAIN_UPTODATE)
 
 client/build/index.html: $(shell find client/app -type f) $(SCOPE_UI_TOOLCHAIN_UPTODATE)
-	mkdir -p client/build
 	if test "true" != "$(SCOPE_SKIP_UI_ASSETS)"; then \
+		rm -rf client/build; \
 		$(SUDO) docker run $(RM) $(RUN_FLAGS) \
 			-v $(shell pwd)/.cache:/home/weave/scope/.cache \
 			-v $(shell pwd)/client:/home/weave/scope/client \
 			-v $(shell pwd)/$(SCOPE_UI_TOOLCHAIN):/home/weave/scope/client/node_modules \
 			-w /home/weave/scope/client \
-			$(SCOPE_UI_BUILD_IMAGE) yarn run build; \
+			$(SCOPE_UI_BUILD_IMAGE) yarn run build-docker; \
+		mkdir -p client/build; \
+		cp -r client/build-docker/* client/build; \
 	fi
 
 client/build-external/index.html: $(shell find client/app -type f) $(SCOPE_UI_TOOLCHAIN_UPTODATE)
-	mkdir -p client/build-external
 	if test "true" != "$(SCOPE_SKIP_UI_ASSETS)"; then \
+		rm -rf client/build-external; \
 		$(SUDO) docker run $(RM) $(RUN_FLAGS) \
 			-v $(shell pwd)/.cache:/home/weave/scope/.cache \
 			-v $(shell pwd)/client:/home/weave/scope/client \
 			-v $(shell pwd)/$(SCOPE_UI_TOOLCHAIN):/home/weave/scope/client/node_modules \
 			-w /home/weave/scope/client \
-			$(SCOPE_UI_BUILD_IMAGE) yarn run build-external; \
+			$(SCOPE_UI_BUILD_IMAGE) yarn run build-external-docker; \
+		mkdir client/build-external; \
+		cp -r client/build-docker-external/* client/build-external; \
```

Comment:
Hmm, I think having 4 different build output dirs might be a bit of an overkill, especially given the docker ones are just temporary in a sense :)

Would your commands with `rm -rf` above be enough to fix #3612? If yes, maybe we can just keep things as simple as possible and only keep those lines, wdyt @imazik?

Reply:
Yes I can clean build directory created by `root` user so that we will have 2 different build dirs only. It will be simple then. :+1: